codeship-steps.yml

```
@@ -61,6 +61,17 @@
   tag: ^(main|develop)
   command: "true"
 
+- name: ga4track_deployment
+  service: ga4tracker
+  tag: ^(main|develop)
+  command: sh -c "export GA_EVENT_NAME=${CI_BRANCH}_CoverAPI && ./ga-event-tracker"
+
+# ${CI_BRANCH} can't be used here because it includes a slash
+- name: ga4track_feature_deployment
+  service: ga4trackerFeature
```

Suggested simplification:
```suggestion
  service: ga4tracker
```

---

Another question (perhaps you've already considered it) would be whether we want to bother tracking feature branches at all. 🤷